tests/test_api/test_endpoints.py

```diff
@@ -173,6 +174,71 @@ def get_token():
         )
 
 
+def test_update_user():
+    mdb = get_mongo(run_config_frozen__normal_env).db
+    rs = ensure_test_resources(mdb)
+    base_url = os.getenv("API_HOST")
+
+    @retry(wait=wait_random_exponential(multiplier=1, max=60))
+    def get_token():
+        """
+
+        Randomly wait up to 2^x * 1 seconds between each retry until the range reaches 60
+        seconds, then randomly up to 60 seconds afterwards
+
+        """
```

Comment:
I added a summary of what the function does, in addition to the retry aspect.

```suggestion
        """
        Fetch an auth token from the Runtime API, retrying until successful.
        
        Randomly wait up to 2^x * 1 seconds between each retry until the range reaches 60
        seconds, then randomly up to 60 seconds afterwards.
        Reference: https://pypi.org/project/retry/
        """
```